frontend/src/routes/(frontend)/[accountId]/index.tsx

```diff
@@ -1,23 +1,131 @@
-import { component$ } from '@builder.io/qwik'
-import { loader$ } from '@builder.io/qwik-city'
+import { $, component$, useStyles$ } from '@builder.io/qwik'
+import { loader$, useNavigate } from '@builder.io/qwik-city'
+import { MastodonAccount } from 'wildebeest/backend/src/types'
+import StickyHeader from '~/components/StickyHeader/StickyHeader'
+import { formatDateTime } from '~/utils/dateTime'
 import { getNotFoundHtml } from '~/utils/getNotFoundHtml/getNotFoundHtml'
+import { formatRoundedNumber } from '~/utils/numbers'
+import styles from '../../../utils/innerHtmlContent.scss?inline'
+import { getAccount } from 'wildebeest/backend/src/accounts/getAccount'
 
-export const accountLoader = loader$(({ request, html }) => {
-	const params = new URL(request.url).searchParams
-	// eslint-disable-next-line @typescript-eslint/no-unused-vars
-	const accountId = params.get('accountId')
+export async function getAccountDetails(
+	domain: string,
+	db: D1Database,
+	accountId: string
+): Promise<MastodonAccount | null> {
+	return await getAccount(domain, accountId, db)
+}
 
-	html(404, getNotFoundHtml())
+export const accountLoader = loader$<{ DATABASE: D1Database }, Promise<MastodonAccount>>(
+	async ({ platform, request, html }) => {
+		const url = new URL(request.url)
+		const domain = url.hostname
+		const accountId = url.pathname.split('/')[1]
 
-	// TODO: retrieve the account details from the backend
-	const accountDetails = null
+		const account = await getAccountDetails(domain, platform.DATABASE, accountId)
 
-	return accountDetails
-})
+		if (!account) {
+			throw html(404, getNotFoundHtml())
+		}
+
+		return account
+	}
+)
 
 export default component$(() => {
-	const accountDetails = accountLoader.use()
+	useStyles$(styles)
+	const nav = useNavigate()
+
+	const accountDetails = accountLoader.use().value
+
+	const goBack = $(() => {
+		if (window.history.length > 1) {
+			window.history.back()
+		} else {
+			nav('/explore')
+		}
+	})
 
-	// TODO: Implement the account view
-	return <>{accountDetails.value && <div>account details</div>}</>
+	const fields = [
+		{
+			name: 'Joined',
+			value: formatDateTime(accountDetails.created_at, false),
+		},
+		...accountDetails.fields,
+	]
+
+	const stats = [
+		{
+			name: 'Posts',
+			value: formatRoundedNumber(accountDetails.statuses_count),
+		},
+		{
+			name: 'Following',
+			value: formatRoundedNumber(accountDetails.following_count),
+		},
+		{
+			name: 'Followers',
+			value: formatRoundedNumber(accountDetails.followers_count),
+		},
+	]
+
+	const accountDomain = getAccountDomain(accountDetails)
+
+	return (
+		<div>
+			<StickyHeader>
+				<div class="flex justify-between items-center xl:rounded-t header bg-wildebeest-700">
+					<button class="text-semi no-underline text-wildebeest-vibrant-400 bg-transparent p-4" onClick$={goBack}>
```

Comment:
Why is this a button and not an anchor?

Reply:
Oh it be because you want to literally do "back"...

Reply:
yeah exactly 🙂 

actually I am tempted to apply this change to the other back anchors as well, since I notice that those point to `/explore` which I feel like may not be the right location in some situations